v1.1.0 on npmjs.com is broken

Error during install:
```
:\#all\FIDATA\workspace\!devops\fidata.org>npm install jjo
npm ERR! path R:\#all\FIDATA\workspace\!devops\fidata.org\node_modules\jjo\dist\lib\jjo.js
npm ERR! code ENOENT
npm ERR! errno -4058
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod 'R:\#all\FIDATA\workspace\!devops\fidata.org\node_modules\jjo\dist\lib\jjo.js'
npm ERR! enoent This is related to npm not being able to find a file.
```
Looking into `jjo-1.1.0.tgz` archive, there is no `dist` dir, just three files: `package.json`, `LICENSE` and `README`

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
 node_modules
 dist
 .vscode
+.devcontainer

--- a/lib/jjo.ts
+++ b/lib/jjo.ts
@@ -1,8 +1,8 @@
 #!/usr/bin/env node
 import commander from "commander";
+import * as fs from "fs";
+import * as path from "path";
 import { parse, parseKeyValuePairs } from "./index";
-
-import * as pkg from "../package.json";
 
 function printJSON(obj: any) {
     if (process.stdout.isTTY) {
@@ -14,6 +14,7 @@ function printJSON(obj: any) {
     }
 }
 
+const pkg = JSON.parse(fs.readFileSync(path.join(__dirname, "../package.json")).toString());
 commander.version(pkg.version);
 commander.description("a small utility to create JSON objects");
 commander.arguments("[items...]");

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "url": "https://github.com/memoryhole/jjo.git"
   },
   "license": "MIT",
-  "main": "dist/lib/index.js",
+  "main": "dist/index.js",
   "scripts": {
     "test": "jest",
     "lint": "tslint -c tslint.json lib/**/*.ts",
@@ -17,7 +17,7 @@
     "clean": "shx rm -rf dist"
   },
   "bin": {
-    "jjo": "dist/lib/jjo.js"
+    "jjo": "dist/jjo.js"
   },
   "lint-staged": {
     "*.ts": "tslint -c tslint.json --fix"

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -12,7 +12,7 @@
     //"sourceMap": true,                     /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
     "outDir": "./dist",                        /* Redirect output structure to the directory. */
-    "rootDir": ".",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+    "rootDir": "./lib",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
     // "composite": true,                     /* Enable project compilation */
     // "removeComments": true,                /* Do not emit comments to output. */
     // "noEmit": true,                        /* Do not emit outputs. */
@@ -56,7 +56,7 @@
     /* Experimental Options */
     // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
-    "resolveJsonModule": true
+    //"resolveJsonModule": true
   },
   "include": [
     "lib/**/*.ts"

